worker/src/RTC/SvcConsumer.cpp

```diff
@@ -586,6 +586,7 @@ namespace RTC
 				MS_DEBUG_TAG(rtp, "sync key frame received");
 
 			this->rtpSeqManager.Sync(packet->GetSequenceNumber() - 1);
+			this->encodingContext->SyncRequired();
```

Comment:
Why is this added? It doesn't seem really related to changes in this PR. Is it a missing call?

@jmillan can you confirm please?

Reply:
I believe this is OK, so we don't re-use any previous PictureId and completely start from a new one.

Is that the intention @jcague?

Reply:
It's not exactly like that, but you are close. SvcConsumer uses the SeqManager only to know if a PictureID is old (it doesn't translate it as SimulcastConsumer does). The isOldPacket gets calculated with the MaxInput value inside the SeqManager. And it looks like the right way to initialize MaxInpu is by calling SeqManager::Sync().

This is what I inferred from the code yesterday. Do you think it makes sense?

Reply:
It makes sense to me